WebRoot/public/extuserprov.jsp

```diff
@@ -67,7 +67,11 @@
 				<table>
 					<tr>
 						<td><app:img id="ZLoginErrorIcon" altkey='ALT_ERROR' src="dwt/ImgCritical_32.png" /></td>
-						<td id="errorMessage"></td>
+						<td id="errorMessage">
+							<% if (request.getAttribute("errorCode") != null) { %>
+								<fmt:message bundle="${zhmsg}" key='${errorCode}'/>
```

Comment:
no issues in using this bundle just wanted to make sure we are covering all error codes :)

as per PR: https://github.com/Zimbra/zm-mailbox/pull/184/files, i can see server can send two error codes 
1) ACCOUNT_EXISTS, this is already present in ZhMsg bundle
2) NO_SHARE_EXISTS, i think this is new error code added, so we probably don't have it in any bundles

@rohan-ambasta let us know, which other error codes we can get in this jsp file besides above two.